pkg/image/controller/signature/container_image_downloader.go

```diff
@@ -46,7 +50,8 @@ func (s *containerImageSignatureDownloader) DownloadImageSignatures(image *image
 
 	signatures, err := source.GetSignatures(ctx)
 	if err != nil {
-		return nil, err
+		glog.V(4).Infof("Failed to get signatures for %v due to: %v", source.Reference(), err)
+		return []imageapi.ImageSignature{}, GetSignaturesError{err}
```

Comment:
just return nil?

Reply:
I followed the pattern you established in this function.